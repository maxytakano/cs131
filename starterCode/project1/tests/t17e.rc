int** w;
int*** x;
int*** y;

function : void main()
{
  if (nullptr == nullptr) {
    x++ = y;
  } else {
    y = x--;
  }

  if (nullptr != nullptr && x++ == y--) {
    w = x + y;
  }
}
