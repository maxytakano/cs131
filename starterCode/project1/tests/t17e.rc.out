Error, "/tmp/cs131_testrunner_KITaux/file.rc", line 8: 
  Left-hand operand is not assignable (not a modifiable L-value).
Error, "/tmp/cs131_testrunner_KITaux/file.rc", line 14: 
  Incompatible type int*** to binary operator +, numeric expected.
Compile: failure.
